File: test/test1/test1.cpp
#include <iostream>
#include <stdio.h>
#include <algorithm>
using namespace std;
int main()
{
    int n=10;
    int num[11];
    int flag=1;
    for(int i=0;i<n;i++)
    {
        scanf("%d",&num[i]);
        if(num[i]<=60)
        {
            flag=0;
        }
    }
    if(flag==1)
        cout<<60<<endl;
    else
    {
        sort(num,num+n);
        int ans;
        ans=num[4]-num[4]%10;
        cout<<ans<<endl;
    }

}

File: test/test1/test1.py
ll = []
flag = 0
sum = 0
for i in range(10):
	out='score'+ str(i+1) +':'
	a = int (raw_input(out))
	ll.append(a)
	sum = sum + a
	if a <= 60:
		flag = 1
ll.sort()
if flag == 0:
	print 60
else:
	print ll[4]-ll[4]%10

